src/styles/base.css

```diff
@@ -454,10 +460,15 @@ header {
     margin: 0;
 }
 
+
 @media screen and (min-width: 992px) {
     .w-md-50 {
         width: 50% !important;
     }
+
+    .mb-lg-6 {
+        margin-bottom: calc(96rem / 16) !important;
+    }
```

Comment:
Could be removed now I think @angela-tran

Reply:
Sounds good, I'll do it in one of the next PRs and link it here